spec/models/event_spec.rb

```diff
@@ -0,0 +1,32 @@
+# spec/models/event_spec.rb
+require 'rails_helper'
+
+RSpec.describe Event, type: :model do
+  subject { described_class.new(event) }
+
+  describe 'valid model' do
+    let(:event) { FactoryGirl.attributes_for(:event) }
+    it { is_expected.to be_valid }
+  end
+
+  describe 'name' do
+    context 'when nil' do
```

Comment:
If we have a context for one direction we'll want an example for the other direction.